packages/react-data-grid/src/masks/InteractionMasks.tsx

```diff
@@ -600,8 +600,8 @@ export default class InteractionMasks<R> extends React.Component<InteractionMask
     const column = columns[draggedPosition.idx];
     const value = getSelectedCellValue({ selectedPosition: draggedPosition, columns, rowGetter });
     const cellKey = column.key;
-    const fromRow = rowIdx < overRowIdx ? rowIdx : overRowIdx;
-    const toRow = rowIdx > overRowIdx ? rowIdx : overRowIdx;
+    const fromRow = rowIdx;
+    const toRow = overRowIdx;
 
     onGridRowsUpdated(cellKey, fromRow, toRow, { [cellKey]: value }, UpdateActions.CELL_DRAG);
```

Comment:
@5joseph9 since we're just renaming variables, we could simply use them in the `onGridRowsUpdated` call,

```suggestion
    onGridRowsUpdated(cellKey, rowIdx, overRowIdx, { [cellKey]: value }, UpdateActions.CELL_DRAG);
```